### PR TITLE
Livestock crates now have 10x more air capacity

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
## About the PR

So that cows stop dying roundstart of suffocation

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: eclips_e
- fix: NT has extended the air tanks located inside of livestock crates, meaning livestock wont suffocate within the first 5 minutes of the shift!

### CODE REVIEW

**Comment by DukeVanity** [comment]
Holy cow! Someone finaly noticed this issue!
I hope that i will get merged

**Comment by moonheart08** [comment]
Why not just give these crates a much larger internal capacity? Their sprite has a visible O2 tank on it.

**Comment by mirrorcult** [comment]
these are 100% supposed to be airtight, just give them a 10x higher capacity

**Comment by Just-a-Unity-Dev** [comment]
> these are 100% supposed to be airtight, just give them a 10x higher capacity

alr

**Review by AJCM-git — CHANGES_REQUESTED** [review]
what mirror said

File: Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml [hunk 1]
```diff
@@ -285,7 +285,7 @@
   - type: StaticPrice
     price: 125
   - type: EntityStorage
-    airtight: false
+    capacity: 5000
```
Comment (DrSmugleaf) [inline Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml:288]: This is the capacity for entities, not air. You would need to make Air in EntityStorageComponent a data field and modify its volume in the yaml.

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml (hunk 1 above)
Comment (Just-a-Unity-Dev) [reply Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml:288]: waz

**Comment by AJCM-git** [comment]
@Just-a-Unity-Dev you coming back to this?

**Comment by Just-a-Unity-Dev** [comment]
Bit busy with school atm so ill close for now

**Comment by metalgearsloth** [comment]
is that what stationware is called now

**Comment by EmoGarbage404** [comment]
> is that what stationware is called now

:godo:
(Play stationware now)

**Pull request closed without merging**